pkg/oauth/server.go

```diff
@@ -65,9 +65,16 @@ type UIConfig struct {
 	FrontendConfig     `name:",squash"`
 }
 
+// APIConfig for upstream APIs.
+type APIConfig struct {
```

Comment:
We may want to move `APIConfig` to `pkg/webui` instead of copy-pasting it from `console` to `oauth`.